packages/sentinel/README.md

```diff
@@ -373,6 +373,16 @@ await client.pause('8181d9e0-88ce-4db0-802a-2b56e2e6a7b1');
 await client.unpause('8181d9e0-88ce-4db0-802a-2b56e2e6a7b1');
 ```
 
+### List Networks
+
+To list tenant enabled networks, you can call the `listNetworks` function on the client, which returns a `Network[]` object:
+
+```js
+await client.listNetworks(); // lists all networks
+await client.listNetworks('prod'); // lists only production networks
```

Comment:
late to the game, but in crypto world do users use terms mainnet and testnet instead of prod and test?

Reply:
Want to be consistent with our UI where we also refer to production and test networks. Also mainnet could be confused with _just_ homestead, whereas we would be referring to all mainnet networks.